### ISSUE TITLE
Clean up env

### ISSUE THREAD

**Issue by junjizhi** [statechannels/monorepo#1317]
**Problem description :confused:**
Current flags in env vars is a bit magic when we want to disable some features. 

**Describe the solution you'd like :rocket:**
Try using https://github.com/niftylettuce/dotenv-parse-variables

**Comment by junjizhi** [comment]
@geoknee what we just discussed

**Comment by geoknee** [comment]
Perhaps we should rename `AUTO_FUND_LEDGER` to `USE_VIRTUAL_FUNDING` to match the wallet?

**Comment by andrewgordstewart** [comment]
> Perhaps we should rename `AUTO_FUND_LEDGER` to `USE_VIRTUAL_FUNDING` to match the wallet?

`USE_VIRTUAL_FUNDING` will be deprecated as of  https://github.com/statechannels/monorepo/pull/1357. 

`AUTO_FUND_LEDGER` should also be deprecated -- web3torrent (any app in general) needs to ask the wallet to approve a budget, and once that is done, `AUTO_FUND_LEDGER` is no longer needed.

**Comment by geoknee** [comment]
Is that lasr point also true for an app wanting to direct fund any channels? Couldn't it skip the budget step entirely ?

**Comment by andrewgordstewart** [comment]
Yes: the whole point of `AUTO_FUND_LEDGER` is that there's no way (that I'm aware of) to fund a ledger from either web3torrent or RPS. As I wanted to test virtual funding in web3torrent and RPS, I needed a way to fund a ledger.

**Comment by kerzhner** [comment]
Closing in favor of https://github.com/statechannels/monorepo/issues/1488
